From f1bde12e3c7ecea19ffe55a510def46db0868837 Mon Sep 17 00:00:00 2001
From: Massimiliano Torromeo <massimiliano.torromeo@gmail.com>
Date: Mon, 4 Nov 2013 16:49:52 +0100
Subject: [PATCH] Fix ssl_wrap_socket AttributeError on python2

Importing requests on python2 raises an AttributeError in pyopenssl.py
---
 requests/packages/urllib3/connectionpool.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/requests/packages/urllib3/connectionpool.py b/requests/packages/urllib3/connectionpool.py
index 1e58143..d347770 100644
--- a/requests/packages/urllib3/connectionpool.py
+++ b/requests/packages/urllib3/connectionpool.py
@@ -42,6 +42,7 @@
     get_host,
     is_connection_dropped,
     Timeout,
+    ssl_wrap_socket,
 )
 
 
-- 
1.8.4

